packages/atoms/src/ListItem/listItemTypes.ts

```diff
@@ -0,0 +1,11 @@
+import { IconType } from '../Icon/iconTypes';
+import { NativeStyleObject } from '@trezor/styles';
+
+export type BaseListItem = {
+    icon?: IconType;
+    title: string;
+    subtitle?: string;
```

Comment:
```suggestion
    subTitle?: string;
```
Use camel case as this invokes some kind of translation to me 😅

Maybe we could use description as name of this props, but with camel case, subTitle is fine

Reply:
https://github.com/trezor/trezor-suite/pull/5482/commits/24f454756b3378827121a7f7b337d6584d90292a